.github/workflows/run_quic_interop_server.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   run_quic_interop_openssl_client:
     strategy:
       matrix:
-        tests: [http3, transfer, handshake, retry, chacha20, resumption, amplificationlimit]
+        tests: [http3, transfer, handshake, retry, chacha20, resumption, amplificationlimit, multiplexing]
```

Comment:
Is it irrelevant for the server runs?

Reply:
you mean the clients?  Yes, the amplification limit is only enforced on the server, and this pipeline tests the openssl server against a series of other client implementations.  Apologies, the name of the pipeline is misnamed, I'll fix that in a subsequent pr

Reply:
Are we really sure the change arrives to correct ?pipeline? there is a `run_quic_interop_server` at line 38 which seems to be more consistent with pr subject.

the other thing is that both those targets contain `amplificationlimit` test. I think this should be relevant for server testing only. However I admit I might be missing some important details here.

Reply:
hmm, you're right, the amplification limit doesn't hurt to do when testing the openssl client against other servers, but its not really necessecary, as the test only confirms that the server obeys the limit.  I'll remove it